app/models/suscribir/suscriptor_anonimo.rb

```diff
@@ -1,11 +1,12 @@
-# coding: UTF-8
-
 module Suscribir
   class SuscriptorAnonimo
     attr_accessor :email, :nombre_apellidos, :cod_postal, :provincia_id
```

Comment:
@Soluciones/devs , it's usual to see things to improve on a PR review, BUT if it's previous code, don't ask the PR author to fix it; you should either fix yourself or add a technical debt card. Otherwise, PRs can stagnate for a too long time...